### PR TITLE
add crux-toolkit

### PR DESCRIPTION
**Pull request by jvolkening**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/crux-toolkit/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  summary: A cross-platform suite of analysis tools for interpreting protein mass spectrometry data
+  license: Apache-2.0
+  license_family: Apache
+  dev_url: https://github.com/crux-toolkit/crux-toolkit
+
+source:
+  url: https://noble.gs.washington.edu/crux-downloads/crux-{{ version }}/crux-{{ version }}.Linux.i686.zip    # [linux32]
+  url: https://noble.gs.washington.edu/crux-downloads/crux-{{ version }}/crux-{{ version }}.Linux.x86_64.zip  # [linux64]
+  url: https://noble.gs.washington.edu/crux-downloads/crux-{{ version }}/crux-{{ version }}.Darwin.x86_64.zip # [osx]
+  sha256: 0cef62d754f432e78ee4df4e5a0e197617ecd337f68eacc16bddda0a8275a5d8 # [linux32]
```
Comment (bgruening) [inline recipes/crux-toolkit/meta.yaml]: Linux32 is not needed. I assume there is no easy way to compile it?

Comment (jvolkening) [reply recipes/crux-toolkit/meta.yaml]: The source is available -- I will revise it to build from that.

Comment (jvolkening) [reply recipes/crux-toolkit/meta.yaml]: There is some weirdness where the build script expects a git branch/revision but it's not being built from a git repo, which throws an exception. In short, building from the source tarball appears non-trivial without some patching.

Is it worth the effort?

**Comment by bgruening** [comment]
Thanks @jvolkening!

**Pull request merged**